src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -147,6 +149,15 @@ private SwiftCompileStep makeCompileStep() {
       compilerCommand.add(
           "-import-objc-header",
           getResolver().getRelativePath(bridgingHeader.get()).toString());
+
+      // bridging header needs exported headers for imports
+      Path headerPath = CxxDescriptionEnhancer.getHeaderSymlinkTreePath(
+          getProjectFilesystem(),
+          BuildTargets.unflavored(getBuildTarget()),
```

Comment:
@ryu2 yeah. Doesn't work. I also saw it while testing but didn't quickly find a way to fix it. Ideally I should somehow just get rid of extra flavors(https://github.com/facebook/buck/blob/master/src/com/facebook/buck/apple/AppleTestDescription.java#L174) instead of hardcoding a check on AppleTestLibrary. 

Do you have any suggestion how can I do it in a clear fashion?